### ISSUE TITLE
Board Reports: Text searches return error ... sometimes

### ISSUE THREAD

**Issue by shrayshray** [datamade/la-metro-councilmatic#560]
Weird queries below, just trying to QA from every angle, okay? :)

Entering a manual search query (maybe only when it does not match a Semaphore term?) on the upgrade test site returns an Internal Server Error. 
[Searching for "bat"](https://lametro-upgrade.datamade.us/search/?q=bat&search-all=on), for example, returns this error:

![image](https://user-images.githubusercontent.com/28063988/75598434-7c72c000-5a50-11ea-8faf-54a421632500.png)

Search for[ "bat" on board agendas](https://boardagendas.metro.net/search/?q=bat&search-all=on) returns 2 results no error.

What's weird is when I do a manual text search on the test upgrade site for something which has no Semaphore term match and no text match in a board report - ["caz", for example](https://lametro-upgrade.datamade.us/search/?q=caz&search-all=on), I get no error; just the expected zero results message:
![image](https://user-images.githubusercontent.com/28063988/75598502-e2f7de00-5a50-11ea-9cd1-e353536e9357.png)
And when I [search for "caz" on board agendas](https://boardagendas.metro.net/search/?q=caz&search-all=on) returns same result - 0 board reports.

**Comment by hancush** [comment]
Love the thorough review, @shrayshray. 

This was a tricky one, but I identified this issues in #564 and #565. The short of it is we needed to update references to board report IDs and attachment text for search results where the query matches attachment text, but not the text of the board report itself – which explains why you were seeing it for some queries, but not others! 

Results for "bat" [are now displaying as expected](https://lametro-upgrade.datamade.us/search/?q=bat&search-all=on).
